### ISSUE TITLE
MessageTicketNumber is triggering for commits with an issue reference using "part of"

### ISSUE THREAD

**Issue by tombruijn** [lintje/lintje#51]
## What happened?

How I called the Lintje command:
```sh
$ lintje
```

The output I received:
```
Hint[MessageTicketNumber]: The message body does not contain a ticket or issue number
  6eba4e2:24:1: Subject
     |
  22 | Part of https://github.com/org/repo/issues/123
  23 |
  24 | Fixes #123
     | ++++++++++ Consider adding a reference to a ticket or issue
     |
     = help: https://r.lintje.dev/r/MessageTicketNumber

1 commit and branch inspected, 0 errors detected, 1 hint
```

## What was supposed to happen?

It shouldn't have suggested adding a reference to an issue. It already had a reference with the "Part of" line.

### Version

```
$ lintje --version
lintje 0.11.0
aarch64-apple-darwin
```

### What Operating System are you seeing the problem on?

macOS, Linux distribution

### How are you running Lintje?

Directly on host machine (Default)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
